src/pages/api/featuredItem.ts

```diff
@@ -16,20 +15,27 @@ const featuredItemDataHandler = async (
     //pass query param to featuredImageID function to check if it is legit
     // logger.info("Generating featured image id");
 
-    const imageID = (request.query.imageID as string)
-      ? featuredImageID(request.query.imageID as string)
-      : featuredImageID();
-
-    const dataFromUri = await getItemDataFromImageID(imageID);
+    const featuredImageData = await getFeaturedImage();
     const numDigitizedItems = await getNumDigitizedItems();
 
     const featuredItemObject = {
-      imageID: imageID,
-      imageSrc: imageURL(imageID),
-      uuid: dataFromUri.uuid,
-      title: dataFromUri.title,
+      imageID: featuredImageData.imageID,
+      backgroundImageSrc: imageURL(
+        featuredImageData.imageID,
+        "full",
+        "!900,900",
+        "0"
+      ),
+      foregroundImageSrc: imageURL(
+        featuredImageData.imageID,
+        "full",
+        "!1100,1100",
+        "0"
+      ),
```

Comment:
Since devops made updates to the memory on the load balancer in front of the IIIF server this morning, images seem to be loading much more quickly/consistently than they were previously.  As a result, I increased the resolution for both the background and foreground images above.  Everything is loading as expected in the Vercel app, but I can't tell if the increased resolution is slowing down the page load slightly.  The latest app is [here](https://digital-collections-756ps0wbe-nypl.vercel.app/), and if you wouldn't mind giving it a try, that would be great!  I can always reduce image resolution back down to what it was before (`700,700` and `900,900`) if needed.